x-pack/plugins/security_solution/public/risk_score/components/enable_risk_score/translations.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import type { RiskScoreEntity } from '../../../../common/search_strategy';
+import { getRiskEntityTranslation } from '../translations';
+
+export const ENABLE_RISK_SCORE_POPOVER = i18n.translate(
+  'xpack.securitySolution.riskScoreDisabled.hostsRiskDashboard.enableRiskScorePopoverTitle',
```

Comment:
Please delete occurrences of the word `host` in this file. Like here: `hostsRiskDashboard`